lib/ui/buttons/IconButton.tsx

```
@@ -1,106 +1,217 @@
-import { getColor, matchColor } from '@lib/ui/theme/getters'
+import { horizontalPadding } from '@lib/ui/css/horizontalPadding'
+import { Spinner } from '@lib/ui/loaders/Spinner'
+import { Size } from '@lib/ui/props'
+import { getColor } from '@lib/ui/theme/getters'
 import { match } from '@lib/utils/match'
-import { ComponentProps } from 'react'
+import { FC, HTMLAttributes } from 'react'
 import styled, { css } from 'styled-components'
+import { Tooltip } from '@lib/ui/tooltips/Tooltip'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)